app/controllers/devise_token_auth/concerns/set_user_by_token.rb

```diff
@@ -113,6 +126,18 @@ def update_auth_header
     end
   end
 
+  def get_cookie_domain
+    # Most people will just set a string for the domain attribute config. But if you need
+    # more flexibility, such as to dynamically choose the domain when serving multiple domains
+    # from a single server, you can set a Proc that will be passed the request.
+    config_value = DeviseTokenAuth.cookie_attributes[:domain]
+    if config_value.is_a?(String)
```

Comment:
These cases are unreached in specs but that could be ok

Reply:
Moot now that I've gutted the unnecessary Proc logic.